build_tools/fetch_artifacts.py

```diff
@@ -60,7 +60,7 @@
 
 # Importing build_artifact_upload.py
 sys.path.append(str(THEROCK_DIR / "build_tools" / "github_actions"))
-from upload_build_artifacts import retrieve_bucket_info
+from post_build_upload import retrieve_bucket_info
 from _therock_utils.artifacts import ArtifactName
```

Comment:
Let's move `retrieve_bucket_info` into a common library (or utils file) instead of sharing across scripts in this way.

(See also Stella's feedback at https://discord.com/channels/1239631572886491286/1370146945867448431/1405954587126403293)